documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -836,9 +888,9 @@ spec:
       name: <storage_map> <8>
       namespace: <namespace>
   targetNamespace: <target_namespace>
-  vms: <9>
-    - id: <source_vm> <10>
-    - name: <source_vm>
+  vms: 
+    - id: <source_vm> <9>
+      name: <source_vm> <10>
```

Comment:
```suggestion
      - name: <source_vm> <10>
```

the "-" makes a difference here, it means the CLI accepts id OR name, e.g. the user can use id or name

it is very important for CLI users to use then only name, because fetching the id is a complicated process, while using the name is what a CLI user know and can do easily

Reply:
@yaacov See https://issues.redhat.com/browse/MTV-1430. Using the hyphen before both the `id` and `name` of the same VM causes the mogratin to fail.

Reply:
Now i know more about VM names and IDs. Everything left as in current documentation. Minor clarifications will be in another PR.